src/test/resources/mail.properties

```diff
@@ -0,0 +1,5 @@
+USER_NAME_1=seleniumtests@tut.by
+PASSWORD_1=123456789zxcvbn
+
+USER_NAME=EE1VP@yandex.by
+PASSWORD=MarEE1!
```

Comment:
I'm not able to add comment to screenshot files, please do not push them to repo, except you have a plan to do that.